### PR TITLE
removes users from updating a particular user endpoint and add a user…

### PR DESCRIPTION
**Pull request by furebo**
# Description

This PR contains changes in user controller at updating user endpoint and at signin response

## Type of change

- [ X] Bug fix (non-breaking change which fixes an issue)

# How should this be manually tested or reproduced?

- Clone this repo
- Checkout to `bg-fixBug-in-userController-#176843073`
- Run the command `npm i`
- Run the command `npm run test` to run tests locally 
- If you push new changes, the test shall run on travis

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [x] I have added tests that prove that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by thepatrickniyo — APPROVED** [review]
yeah! that will help in integration

File: src/controllers/user.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export const signin = (req, res) => {
           jwt.verify(token, process.env.JWT_KEY, (err, data) => {
            
           });
-          res.json({ success: true, token: `JWT ${token}` });
+          res.json({ success: true, token: `JWT ${token}`,user });
```
Comment (SimonImanigirimpuhwe) [inline src/controllers/user.js]: Consider not returning password in the user info you are returning here.

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]

File: src/controllers/user.js (hunk 1 above)
Comment (furebo) [reply src/controllers/user.js]: The password in user info is encrepted. Is there any virnerability?

Comment (SimonImanigirimpuhwe) [reply src/controllers/user.js]: No matter how it's encrypted, it should not be exposed

**Review by SimonImanigirimpuhwe — APPROVED** [review]

**Review by SamuelSabasNayo — APPROVED** [review]

**Pull request merged**